examples/language-modeling/README.md

```diff
@@ -69,6 +69,27 @@ python run_clm.py \
     --throughput_warmup_steps 3
 ```
 
+To run with streaming dataset, use the '--streaming' flag with '--max_steps' and optionally'--max_eval_samples' (only when evaluation is enabled) specified like so:
+
+```bash
+python run_clm.py \
+    --model_name_or_path gpt2 \
+    --dataset_name wikitext \
+    --dataset_config_name wikitext-2-raw-v1 \
+    --per_device_train_batch_size 4 \
+    --per_device_eval_batch_size 4 \
+    --do_train \
+    --output_dir /tmp/test-clm \
+    --gaudi_config_name Habana/gpt2 \
+    --use_habana \
+    --use_lazy_mode \
+    --use_hpu_graphs_for_inference \
+    --throughput_warmup_steps 3 \
+    --streaming \
+    --max_steps 1000 \
+    --do_eval \
+    --max_eval_samples 200
+```
```

Comment:
Let's move this paragraph into the "Streaming" section of the README: https://github.com/huggingface/optimum-habana/tree/main/examples/language-modeling#streaming